src/cript/nodes/core.py

```diff
@@ -260,6 +261,9 @@ class ReturnTuple:
         previous_handled_nodes = copy.deepcopy(NodeEncoder.handled_ids)
         if handled_ids is not None:
             NodeEncoder.handled_ids = handled_ids
+        previous_known_uuid = copy.deepcopy(NodeEncoder.known_uuid)
```

Comment:
I advise against deepcopy whenever possible just because I had a very hard time when working with fixtures that used it for the integration tests. It just made things more complicated, but if we have to use it then we can use it

Reply:
No, there is a deepcopy necessary here.

However, this is a deepcopy of a list of strings. Not of nodes.
So very different then your issues in the fixtures.

The fixture issues are only caused, because DB schema doesn't allow UID where it should sometimes. We need to fix that separately.

Reply:
okie dokes, that sounds good!